resolwe_bio/docker_images/common/packages-manual/sra-toolkit.sh

```diff
@@ -5,16 +5,16 @@
 download_and_verify \
     ncbi \
     sra-toolkit \
-    2.9.0 \
-    911301a6f85bdbf8e786a6b6561d32f262cd356ae52acd27830e39fb07e41b12 \
+    2.10.0 \
+    0a283d8cf7ff9da9ff836056bd0a7e4c49c583cdc3c440e2c316c4dbb3029c80 \
     https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/\${version}/sratoolkit.\${version}-ubuntu64.tar.gz \
     sratoolkit.\${version}-ubuntu64
 
-mkdir /.ncbi/
-echo '## auto-generated configuration file - DO NOT EDIT ##' >>/.ncbi/user-settings.mkfg
-echo '' >>/.ncbi/user-settings.mkfg
-echo '/config/default = "false"' >>/.ncbi/user-settings.mkfg
-echo '/repository/user/cache-disabled = "true"' >>/.ncbi/user-settings.mkfg
+# create a config file
+mkdir -p /root/.ncbi
+printf '/LIBS/GUID = "%s"\n' `uuidgen` > /root/.ncbi/user-settings.mkfg
```

Comment:
I'm curious to know what was the need for `printf` here over `echo`.

Reply:
`echo` does't expand the output of ``uuidgen`` to `$s` By using printf, I was able to reuse this commands from the sra-toolkit dockerfile. To use echo, I would need to find another way of doing that.